src/controllers/userController.js

```diff
@@ -50,6 +73,17 @@ const showCurrentUser = async (req, res) => {
 
 // Update the information of the current user
 const updateCurrentUser = async (req, res) => {
+  /*
+     #swagger.summary = 'Fetch a user by id, update their data and return a new token user.'
+     #swagger.parameters['id'] = {
+        description: 'Mongo ObjectID of the user to fetch',
+     }
```

Comment:
This endpoint does not take any query or path parameters; it just uses the cookie to find the requesting user's id.